src/main/java/com/global/inbox/service/ItemService.java

```diff
@@ -1,10 +1,13 @@
 package com.global.inbox.service;
 
+import com.global.inbox.dto.ItemDto;
 import com.global.inbox.model.Item;
 
 import java.util.List;
 
 public interface ItemService {
 
+    Item save(ItemDto itemDto);
```

Comment:
If we are locking data/domain service, then let's receive and return only DTO's then